packages/ui/src/components/vuestic-components/va-dropdown/VaDropdown.vue

```diff
@@ -22,9 +22,9 @@
         class="va-dropdown__anchor-width-container"
         :style="anchorWidthContainerStyles"
       >
-        <slot />
+        <slot/>
       </div>
-      <slot v-else />
+      <slot v-else/>
```

Comment:
can you configure ide in a way so that it styles self closing tags the way they are now? Alsom can you check if linter can autofix these?

Reply:
eslint config was extended